src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx

```diff
@@ -23,7 +23,21 @@ interface IGroupComparisonMutationsTabPlotProps {
 }
 
 function plotYAxisLabelFormatter(symbol?: string, groupName?: string) {
-    return `${symbol} ${groupName} Mutations`;
+    // lowercase = 1 and uppercase = 1.3 (based on 'w' and 'W'), if groupName >= 13, stop and + "..."
+    let length = 0;
+    let label = '';
+    for (let c of groupName!) {
```

Comment:
why not take symbol into account too?

Reply:
the symbol is always % or # I believe with a space after so I just need to look at the groupName. I found that 13 'w' and 10 'W' with the symbol + space before it is the max.